CHANGELOG.md

```diff
@@ -9,13 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Support for `AtomicCmpExchange128` intrinsic.
+- Support for `GetTypeKind` intrinsic.
```

Comment:
Should these be "intrinsic routine"? It looks a little odd having both "intrinsic" and "intrinsic type".

Reply:
Hm. Referring to the routines as just "intrinsics" is consistent with other changelog entries. I agree that it's (a little) confusing though since there's a type in the mix this time. 
On balance I like the current wording, but I'd be happy to add "routine" if you want.

Reply:
The changelog entries are inconsistent and vary between `intrinsic`, `intrinsic routine`, and `intrinsic method`. I'm not particularly worried about that. Thoughts on `Support for the <x> intrinsic` rather than `Support for <x> intrinsic`? The former sounds better to me.